testgrid/config.yaml

```
@@ -2843,6 +2843,11 @@ dashboards:
     description: "Uses kubetest to run e2e tests (+Feature:Ingress|NEG) against a cluster created with cluster/kube-up.sh"
     alert_options:
       alert_mail_to_addresses: kubernetes-sig-network-test-failures@googlegroups.com
+  - name: skew-cluster-latest-kubectl-stable1-gce
+    test_group_name: ci-kubernetes-e2e-gce-master-new-gci-kubectl-skew
+    description: stable1 e2e tests run against a master gce cluster using a stable1 kubectl binary
```

Can we be more specific about what this does?
```suggestion
    description: Uses kubetest to run e2e tests (+Kubectl, -Serial) from the previous release of kubernetes (stable1) against a cluster created with cluster/kube.sh from master
```